packages/@aws-cdk/aws-events/lib/event-bus.ts

```diff
@@ -332,13 +338,39 @@ export class EventBus extends EventBusBase {
     this.eventBusPolicy = eventBus.attrPolicy;
     this.eventSourceName = eventBus.eventSourceName;
   }
+
+  public addToResourcePolicy(statement: iam.PolicyStatement): iam.AddToResourcePolicyResult {
+    if (statement.sid == null) {
+      throw new Error('Event Bus policy statements must have a sid');
+    }
+
+    if (this.policy) {
+      // The policy can contain only one statement
+      return { statementAdded: false };
+    }
+
+    if (this.autoCreatePolicy) {
```

Comment:
[nit] In the context of this class alone, it reads strangely to check for an `autoCreatePolicy` in the `if` clause when one is supplied, would it not be better to reference the `import` aspect instead?

Reply:
Oh, you're absolutely right! There's no need for this property at all.